### DISCUSSION TITLE
ECHIPA CARPATII

### DISCUSSION THREAD

**Discussion by sebiboga** [peviitor-ro/community#105] [TEAMs]
Aici sunt membrii echipei CARPATII

**Comment by sebiboga** [comment]
Andrei Cojocaru 
role: scraper developer
discord: return_1101

**Reply by return1101** [reply]
Am nevoie de 3 testeri manuali. Avem?

Andrei, ai interviu cu 2 manual testeri: Danut Bese si Diana Stanculescu
Simina Antip, manual tester

**Reply by sebiboga** [reply]
Dacă nu avem, scoatem open positions in motorul de căutare...deci o să avem...

**Comment by AdrianaCionca** [comment]
Daniela Stanculesc
role: Team Lead and Manual Tester
discord: DanielaStanculesc

**Comment by AdrianaCionca** [comment]
Simina Antip
role: Manual Tester
discord: simina1606

**Comment by AdrianaCionca** [comment]
Elena Istratescu
role: Manual Tester
discord: Elena23

**Comment by return1101** [comment]
Cristi Toargă
role: Scraperist, repo -> Scrapers_start_with_digi (Va lucra pe acest repo).
discord: mafiaaaaaaaaa
Stagiu de pregatire

**Comment by AdrianaCionca** [comment]
Ioana Maria Matei
role: Manual Tester
discord: ioanamariamatei

**Comment by AdrianaCionca** [comment]
Joita Madalina-Mihaela
role: Manual Tester
discord: JoitaMadalina-Mihaela
